app/presenters/dashboard.rb

```diff
@@ -14,6 +14,10 @@ def unsubmitted_exercises
         user.exercises.unsubmitted
       end
 
+      def unsubmitted_grouped_exercises
+        user.exercises.unsubmitted.group_by { |ex| ex.language }
```

Comment:
@mhelmetag What do you think about memoizing this as well as the other functions in this class that return database responses and calculations.  I think it would be great to have this pattern going forward as more developers continue to contribute and make sure everything remains performant.

```
def unsubmitted_grouped_exercises
  @unsubmitted_grouped_exercises ||= user.exercises.unsubmitted.group_by { |ex| ex.language }
end
```